build-ubuntu.sh

```diff
@@ -1,11 +1,79 @@
 #!/bin/bash
+cd "$(dirname ""$0"")"
 
+# some magick from 
+# https://github.com/tiwilliam/vagrant-debian
+set -e
+
+function usage(){
+    echo "usage: $0 <32|64> [LTS] "
+    exit
+}
+
+if [ $# -lt 1 ] ; then
+    usage;
+fi
+
+if [ $# -gt 2 ] ; then
+    usage;
+fi
+
+argv=($@)
+case ${argv[0]} in
+    32)
+        ARCH="i386"
+        OS_TYPE="Ubuntu"
+    ;;
+    64)
+        ARCH="amd64"
+        OS_TYPE="Ubuntu_64"
+    ;;
+    *)
+        usage;
+    ;;
+esac
+
+# start bercut497 magick
+# https://github.com/bercut497/vagrant-box
+LTS=0
+if [ -n "${argv[1]}" ] ; then
+    if [ "${argv[1]}" = "LTS" ] ; then
+        LTS=1
+    else
+        usage ;
+    fi
+fi
+
+if [ "$( date +%m)" -lt 10 ] ;then
+  V_MONTH="04"
+fi
+
+V_YEAR="$( date +%y)"
+if [ "${LTS}" -eq 1 ] ; then
+  echo "LTS SET"
+  if [ "$((${V_YEAR}%2))" -eq 1 ] ; then
```

Comment:
A new LTS version is released every 2 years.
[ref](https://wiki.ubuntu.com/LTS)